src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAudio.cs

```diff
@@ -105,5 +105,99 @@ public void Disconnect(BluetoothAudioProfileType type)
+        public void OpenAgSco()
+        {
+            if (BluetoothAdapter.IsBluetoothEnabled && Globals.IsInitialize)
+            {
+                int ret = BluetoothAudioImpl.Instance.OpenAgSco();
+                if (ret != (int)BluetoothError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to OpenAgSco - " + (BluetoothError)ret);
+                    BluetoothErrorFactory.ThrowBluetoothException(ret);
+                }
+            }
+            else
+            {
+                BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.NotEnabled);
+            }
+        }
+
+        /// <summary>
+        /// Closes a AG(Audio Gateway) SCO(Synchronous Connection Oriented link) to connected remote device, asynchronously.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public void CloseAgSco()
+        {
+            if (BluetoothAdapter.IsBluetoothEnabled && Globals.IsInitialize)
+            {
+                int ret = BluetoothAudioImpl.Instance.CloseAgSco();
+                if (ret != (int)BluetoothError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to CloseAgSco - " + (BluetoothError)ret);
+                    BluetoothErrorFactory.ThrowBluetoothException(ret);
+                }
+            }
+            else
+            {
+                BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.NotEnabled);
+            }
+        }
+
+        /// <summary>
+        /// A property to check whether an opened AG(Audio Gateway) SCO(Synchronous Connection Oriented link) exists or not.
+        /// </summary>
+        /// <exception cref="NotSupportedException">Thrown when the Bluetooth is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        static public bool IsAgScoOpened
+        {
+            get
+            {   
+                return BluetoothAudioImpl.Instance.IsAgScoOpened;
+            }
+        }
+
+        /// <summary>
+        /// This event is called when the AG(Audio Gateway) SCO(Synchronous Connection Oriented link) state is changed.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public event EventHandler<AgScoStateChangedEventArgs> AgScoStateChanged
+        {
+            add
+            {
+                BluetoothAudioImpl.Instance.AgScoStateChanged += value;
+            }
+            remove
+            {
+                BluetoothAudioImpl.Instance.AgScoStateChanged -= value;
+            }
+        }
+
+        /// <summary>
+        /// Notifies the state of AG(Audio Gateway) voice recognition.
+        /// </summary>
+        /// <param name="state">The state of voice recognition.</param>
+        /// <since_tizen> 6 </since_tizen>
+        public void NotifyAgVoiceRecognitionState(bool state)
```

Comment:
What mean state? is it enable and disable?

Reply:
'state' means voice recognition state of AG is enabled or not.
I modified the xml document. Thank you.

Reply:
If it means enable, please also update parameter name to enable like below
``` c#
 public void NotifyAgVoiceRecognitionState(bool enable)
```

Reply:
Done. Thank you.